src/agent/v8/debugapi.ts

```diff
@@ -46,10 +46,15 @@ const nodeVersion = /v(\d+\.\d+\.\d+)/.exec(process.version);
 if (!nodeVersion || nodeVersion.length < 2) {
   const dummyapi = require('./dummy-debugapi');
   debugApiConstructor = dummyapi.DummyDebugApi;
-} else if (semver.satisfies(nodeVersion[1], '>=8')) {
+} else if (
+    semver.satisfies(nodeVersion[1], '>=8') &&
+    process.env.GCLOUD_USE_INSPECTOR) {
   const inspectorapi = require('./inspector-debugapi');
   debugApiConstructor = inspectorapi.InspectorDebugApi;
 } else {
+  if (process.env.GCLOUD_USE_INSPECTOR) {
```

Comment:
Maybe we should move this check to the V8DebugAPI constructor, because there we have access to a logger object. @gaofanmichael @DominicKramer thoughts?

Reply:
I agree